### PR TITLE
fix bid_tendering_mode in purchase requisition

### PR DESCRIPTION
**Pull request by gurneyalex**
set the field required in the model rather than in the view
add a default value for the field

### CODE REVIEW

**Comment by lepistone** [comment]
Thanks! :+1:

File: purchase_requisition_bid_selection/model/purchase_requisition.py [hunk 1]
```diff
@@ -65,6 +65,8 @@ class PurchaseRequisition(models.Model):
         [('open', 'Open'),
          ('restricted', 'Restricted')],
         'Call for Bids Mode',
+        required=True,
+        default = 'open',
```
Comment (LeartS) [inline purchase_requisition_bid_selection/model/purchase_requisition.py]: Nitpicking: to space or not to space? http://legacy.python.org/dev/peps/pep-0008/#other-recommendations

**Comment by PCatinean** [comment]
:+1:

File: purchase_requisition_bid_selection/model/purchase_requisition.py (hunk 1 above)
Comment (lepistone) [inline purchase_requisition_bid_selection/model/purchase_requisition.py]: your comment was not lost :smile_cat: but don't worry: travis is there checking those things.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**